index.html

```diff
@@ -36,5 +36,190 @@
         </div>
       </section>
   </section>
+   <section class="work">
+    
+    
+    
+    <div class="second-page">
+      <div class="cards">
+        <div class="screenshoot">
+          <img src="images/Snapshoot Portfolio (1).png" alt="portifolio screenshoot">
+        </div>
+
+        <div class="description">
+          <h2>Tonic</h2>
+          <span class="part">
+            <ul>
+              <li id="canopy">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </span>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <span class="tech-stack">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </span>
+        <div class="button">
+          <button>see projects</button>
+        </div>
+      </div>
+      </div>
```

Comment:
- [ ] As demanded by project requirements, you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly consider the following;
  - Edit the background color you have on the body to match the one on the figma template.
  - Add a border radius to each work card to match the template
  - Add a border to work card to match template
  - Kindly edit the list of technologies layout and colors for each work card to match template
![image](https://github.com/Etsubdink-Demeke/portifolio/assets/94551484/11be3e3d-ffe5-4dba-8570-a2be317579ee)
  -  Kindly edit this section layout to match design template
![image](https://github.com/Etsubdink-Demeke/portifolio/assets/94551484/93812298-4374-494f-8015-8c36e84ac399)
  - Kindly edit the background color and border of your buttons to match design template
  - Kindly add some space between work cards to match template.
  - I noticed your work card isn't centralized. The space on the left is greater than that on the right. Kindly fix this please.

### Your design
![image](https://github.com/Etsubdink-Demeke/portifolio/assets/94551484/5329e4ae-8b82-43b0-bbce-c1be4737534b)
### Figma
![image](https://github.com/Etsubdink-Demeke/portifolio/assets/94551484/718b0862-c83f-4ea8-b810-be0c17415c3f)